riverdriver/riversqlite/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/riverqueue/river/riverdriver/riversqlite
+
+go 1.23.0
+
+toolchain go1.24.1
+
+require (
+	github.com/riverqueue/river/riverdriver v0.20.2
+	github.com/riverqueue/river/rivershared v0.20.2
+	github.com/riverqueue/river/rivertype v0.20.2
+	github.com/stretchr/testify v1.10.0
+)
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
```

Comment:
Shouldn't this driver module actually require the sqlite library i.e. `modernc.org/sqlite`?

Reply:
So believe it or not ... no. All code in the driver itself only depends on `sqlc` primitives. That's actually the case for `riverdriversql` as well, except that sqlc does need to pull in pq for a couple small things like the `pq.Array` helper.

We need an underlying driver, but only where we're actually running tests, and just the way the project is laid out, it turns out that all the tests are in the top-level package to prevent package dependency cycle problems (e.g. see `driver_test.go`). It would actually be nice to see if we could take another stab at pushing tests into each specific driver maybe, and if we pulled that then this package would pick up the `modernc.org/sqlite` dependency.

Reply:
Also, I ran `go mod tidy` on all these files now, so the dependencies should be a little more correct now.